bin/setup-local-npm-registry.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Should we probably add some npm scripts to execute these files in `package.json`  like we have commands for other `./bin` files?

Reply:
Yeah why not 👍  Will add